component/sections/pricing/variant_a.js

```diff
@@ -99,12 +102,22 @@ function VariantA({ caption, title, description, plans }) {
                   {plans?.[0]?.primaryButton?.label && (
                     <a
                       className="inline-block text-center py-2 px-4 w-full rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose transition duration-200"
-                      target={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                      rel={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                      target={
+                        plans?.[0]?.primaryButton?.linkTarget ===
```

Comment:
This **_target_** attribute value needs to be updated. Need to change this value and as follows accordingly to 
`plans?.[index]?.primaryButton?.linkTarget` .